docs/conf.py

```diff
@@ -104,7 +104,11 @@
 # https://sphinx-simplepdf.readthedocs.io/en/latest/configuration.html
 
 simplepdf_vars = {
-    "primary": "#ffcc00",
     "cover": "black",
+    # Confusing! The white gets applied to the page number color (which appears inside a rusty red
+    # box) and the back cover text color. Neither white nor black work well for the back page text
+    # color since the text intersects the thick black bottom arc of the Pex P-egg. The yolk color
+    # works well enough in both these spots though.
+    "white": "#ffee00",
```

Comment:
Alright, that's it from me. Sorry for the thrash.

The PDF looks good enough. Site re-deployed with this change and the PDF link is the PDF icon on the bottom bar: https://jsirois.github.io/pex/